Vagrantfile

```diff
@@ -56,5 +59,7 @@ setenforce 0
 systemctl stop firewalld
 sudo groupadd docker && sudo gpasswd -a ${USER} docker && sudo systemctl restart docker && newgrp docker
 usermod -aG docker vagrant
+sudo modprobe overlay
+echo "131072" | sudo tee /sys/module/nf_conntrack/parameters/hashsize
```

Comment:
Unfortunately it's necsssary. The hashsize patch is included in K8s 1.7. However kube-proxy doesn't run when hashsize=16k, but it starts to run when hashsize=32k or higher. That's probably because under Fedora 26 in the Vagrant VM, `net.nf_contrack_max` is set to 64k by default. So when the current hashsize isn't greater than (max / 4), say 16k, it doesn't escape from the code above, which results in an error due to netns restriction. That's what I think it's happening.

We could make a more brilliant guess here, but I just took a simple approach to increase the hashsize to 128k.